CONTRIBUTING.md

```diff
@@ -175,5 +175,5 @@ Refer to the [Motr Coding Style Guide](../dev/doc/coding-style.md) and the [CORT
 
 You can reach out to us with your questions, feedback, and comments through our CORTX Communication Channels:
 
-- Join our CORTX-Open Source Slack Channel to interact with your fellow community members and gets your questions answered. [![Slack Channel](https://img.shields.io/badge/chat-on%20Slack-blue)](https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?)
+- Join our CORTX-Open Source Slack Channel to interact with your folk community members and gets your questions answered. [![Slack Channel](https://img.shields.io/badge/chat-on%20Slack-blue)](https://join.slack.com/t/cortxcommunity/shared_invite/zt-femhm3zm-yiCs5V9NBxh89a_709FFXQ?)
```

Comment:
I think we can change it to :
join our CORTX-Open Source Slack Channel to interact with other community members and get your questions answered